src/main/kotlin/uk/gov/justice/hmpps/offendersearch/controllers/OffenderSearchController.kt

```diff
@@ -248,4 +249,52 @@ class OffenderSearchController(
   ): SearchPagedResults {
     return searchService.findByListOfTeamCodes(pageable, teamCodeList)
   }
+
+  @PreAuthorize("hasRole('ROLE_COMMUNITY')")
+  @ApiOperation(value = "Match prisoners by a ldu code", notes = "Requires ROLE_COMMUNITY role")
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        code = 200,
+        message = "OK",
+        response = SearchPagedResults::class
+      ), ApiResponse(
+        code = 400,
+        message = "Invalid Request",
+        response = BadRequestException::class
+      ), ApiResponse(code = 404, message = "Not found", response = NotFoundException::class)
+    ]
+  )
+  @GetMapping("/ldu-codes/{lduCode}")
+  fun searchOffendersByLduCode(
+    @PathVariable lduCode: String,
+    @PageableDefault pageable: Pageable,
+  ): SearchPagedResults {
+    log.info("Search called with {}", lduCode)
```

Comment:
This looks a lot like the team code search info log - might be worth differentiating them.